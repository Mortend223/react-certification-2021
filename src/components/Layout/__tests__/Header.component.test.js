import HeaderComponent from '../Header.component';

describe('Header Component', () => {
    it('Should show HeaderComponent', () => {
        
    });
});