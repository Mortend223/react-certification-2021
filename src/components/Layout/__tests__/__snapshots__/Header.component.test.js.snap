// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should show HeaderComponent 1`] = `undefined`;
